Compiler/src/precompile.jl

```diff
@@ -423,7 +423,7 @@ function compile_and_emit_native(worlds::Vector{UInt},
         isa(exc, Core.TrimFailure) || rethrow()
         # The verification check failed. The error message should already have
         # been printed, so give up here and exit (w/o a stack trace).
-        exit(1)
+        invokelatest(exit(1))
```

Comment:
Shouldn't this be
```suggestion
        invokelatest(exit, 1)
```
?